core/src/test/java/org/elasticsearch/indices/store/IndicesStoreIntegrationTests.java

```diff
@@ -143,6 +158,10 @@ public void indexCleanup() throws Exception {
 
     }
 
+    private void moveShardFromNode1ToNode3(String node_1, String node_3) {
```

Comment:
can inline this method? it's just a one liner and the name is kind of funky - node 1 and node 3 are baked into it.